.github/actions/install-rust/action.yml

```diff
@@ -5,7 +5,7 @@ inputs:
   toolchain:
     description: 'Default toolchan to install'
     required: false
-    default: 'stable'
+    default: '1.41.0' # currently limited by Firefox's minimum rustc version.
```

Comment:
This will probably make the `peepmatic` jobs fail, right?

In the "checks" CI job, could we do:

```
rustup install 1.41.0
cargo +1.41.0 check -p cranelift
```

?

That would check just the minimum necessary without affecting the rest of the code base, I think.

Reply:
Oh, that's a better idea than running all tests redundantly. Thanks!